addon/components/modal.js

```diff
@@ -41,27 +33,11 @@ export default class ModalComponent extends Component.extend({
 	init() {
 		super.init(...arguments);
 
-		// Prevent creating an uncaught promise.
-		this.model.promise.catch(() => {}).finally(
-			this._close.bind(this),
-			`Component '${this.model.fullname}': close modal`
-		);
-	}
-
-	didInsertElement() {
-		super.didInsertElement(...arguments);
-
-		run.next(this.scheduler, 'scheduleOnce', this, '_open');
+		next(this, '_open');
```

Comment:
This must be called in `didInsertElement` hook to maintain the functionality

Reply:
Done!